zerver/openapi/zulip.yaml

```diff
@@ -7134,8 +7134,26 @@ paths:
                           type: integer
                         description: |
                           An array with the IDs of the modified messages.
+                      skipped_marking_unread_stream_ids:
```

Comment:
perhaps rename it to `unread_streams` ?
in future we might do other things as well, not just mark skipped.

Better to discuss this naming once in CZO

Reply:
Apparently there was a [CZO discussion](https://chat.zulip.org/#narrow/stream/378-api-design/topic/Not.20marked.20unread.20banner/near/1637370)where the name was decided. Updated the code.